src/operations.jl

```diff
@@ -9,25 +9,26 @@ relation(db::DB, synset::Synset, pointer_sym) = map(
     filter(ptr -> ptr.sym == pointer_sym, synset.pointers)
 )
 
-antonyms(db::DB, synset::Synset)  = relation(db, synset, ANTONYM)
-hyponyms(db::DB, synset::Synset)  = relation(db, synset, HYPONYM)
+function expanded_relation(db::DB, synset::Synset, pointer_sym)
+    all_related = Set{Synset}([synset])
```

Comment:
```suggestion
    # Inititally include self, so as to avoid cycles,
    # but we will delete it at the end 
    all_related = Set{Synset}([synset])
```